decisions/0015-env-triad-terms.md

```diff
@@ -1,52 +1,124 @@
+  are added to existing classes.
 
-To ensure interoperability and consistancy, the NMDC General Query will be ammended for each environmental sample type.
-- To satisfy NMDC needs, we will start here with soil.
-- This ADR will be updated as environment specific queries are created.
-- No environment specific query should be "cherry picked". Rather any filtering should be accomplished as a general query. (Example, will not remove a specific term by ID, but rather identify what is the rule that can be created to fit a overall need.)
+## NMDC General Queries for Environmental Triad Value Sets
 
-NMDC General Query
-- `env_broad_scale` will be composed of terms that branch from biome [ENVO:00000428]
-  - We will then evaluate what is lost when ecosystem is excluded and determine what needs “re-added” or requested of envo
-- `env_local_scale` will be composed of terms that branch from material entity [BFO:0000040], excluding biome [ENVO:00000428] & environmental material [ENVO:00010483]
-  - Additional narrowing will occur to remove specific branches. 
-- `env_medium` will be composed of terms that branch from environmental material [ENVO:00010483]
+- `env_broad_scale` will consist of subclasses
+  of [biome [ENVO:00000428]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000428),
+  with the exception of host-associated samples, including plant-associated samples.
+- `env_local_scale` will consist of subclasses
+  of [material entity [BFO:0000040]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FBFO_0000040?lang=en),
+  minus biome [ENVO:00000428]
+  and [environmental material [ENVO:00010483]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00010483)
+    - It's expected that additional branches of the is_a hierarchy will subtracted,
+      like [chemical entity [CHEBI:24431]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCHEBI_24431)
+    - The possibility of subtracting leaf nodes is under consideration. Since we are programmatically building candidate
+      tables with multiple forms of evidence, the smaller query result offered by subtraction may not be of much added
+      benefit
+    - Subtraction over relationships other than is_a, subtraction of classes based on substrings in their labels, or
+      other targets subtraction are possible, but may also be of limited benefit
+    - @turbomam
+      has [prototyped these workflows in Makefiles](https://github.com/microbiomedata/external-metadata-awareness/blob/main/Makefiles/soil-env_local_scale.Makefile).
+      @sierra-moxon will explore options for making the workflows more accessible to the whole group, with a single
+      application that takes a single configuration file, or with (a) Jupyter notebook(s)
```

Comment:
```suggestion
 
```

Not what the ADR is about. Exploration, possibilities, and options aren't a decision. Leave out until we decide.